### PR TITLE
makes view for video content responsive for tablets

### PR DESCRIPTION
**Pull request by jtamiace**
still need to implement Modernizer for best use/cooperation of Javascript and CSS

### CODE REVIEW

File: kalite/distributed/templates/distributed/learn.html [hunk 1]
```diff
@@ -27,10 +27,10 @@
         {% compress css file perseuscss %}
         <link rel="stylesheet" type="text/css" href="{% static 'js/distributed/perseus/ke/css/khan-exercise.css' %}">
         <link rel="stylesheet" type="text/css" href="{% static 'css/distributed/video-js.css' %}">
-
         <link rel="stylesheet" type="text/css" href="{% static "js/distributed/perseus/lib/katex/katex.min.css" %}" />
         <link rel="stylesheet" type="text/css" href="{% static "js/distributed/perseus/lib/mathquill/mathquill.css" %}" />
         <link rel="stylesheet" type="text/css" href="{% static "js/distributed/perseus/build/perseus-3.css" %}" />
+        <link rel="stylesheet" type="text/css" href="{% static 'css/distributed/video-js-override.css' %}">
```
Comment (radinamatic) [inline kalite/distributed/templates/distributed/learn.html]: This CSS file looks like a better place to put my focus changes from #4190. @rtibbles ?

Comment (rtibbles) [inline kalite/distributed/templates/distributed/learn.html]: Indeed.

**Comment by rtibbles** [comment]
Merging as is, as Modernizr depends on changes in another PR.

File: kalite/distributed/templates/distributed/learn.html (hunk 1 above)
Comment (radinamatic) [inline kalite/distributed/templates/distributed/learn.html]: Done.

**Pull request merged**